common/defines/babylon_defines.txt

```diff
@@ -1,40 +1,97 @@
-	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.05 #0.1	# War Exhaustion from casualties is always multiplied by at least this
-	WAR_EXHAUSTION_KIA_BATTLES_LOST_MULTIPLIER = 0.35 #0.25	# At 100% of battles lost by our side, War Exhaustion from casualties is further multiplied by this 
-	WAR_EXHAUSTION_TURMOIL_FACTOR = 2.5 #2 At 100% turmoil
-	WAR_EXHAUSTION_CONTESTED_ENEMY_WARGOALS = 3.0 #1
+	WAR_EXHAUSTION_KIA_FACTOR = 5	#20 # Scaled by casualties compared to total manpower
+	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.05	#0.1	# War Exhaustion from casualties is always multiplied by at least this
+	WAR_EXHAUSTION_KIA_BATTLES_LOST_MULTIPLIER = 0.35	#0.25	# At 100% of battles lost by our side, War Exhaustion from casualties is further multiplied by this 
+	WAR_EXHAUSTION_TURMOIL_FACTOR = 2.5	#2 At 100% turmoil
+	WAR_EXHAUSTION_CONTESTED_ENEMY_WARGOALS = 3	#1
 	OCCUPATION_STATE_BASE_WEIGHT = 1
 	OCCUPATION_STATE_POP_WEIGHT = 1
 	OCCUPATION_STATE_INCORPORATED_WEIGHT = 10
-	
 	STATE_CONQUEST_HOMELAND_AND_CLAIM_RADICALIZATION = 0.1
 	STATE_CONQUEST_HOMELAND_AND_CLAIM_DEMILITARIZATION = 0.25
 	STATE_CONQUEST_HOMELAND_OR_CLAIM_RADICALIZATION = 0.2
 	STATE_CONQUEST_HOMELAND_OR_CLAIM_DEMILITARIZATION = 0.5
 	STATE_CONQUEST_DEFAULT_RADICALIZATION = 0.4
 	STATE_CONQUEST_DEFAULT_DEMILITARIZATION = 0.75
+	NUM_PROVINCES_FOR_ADVANCEMENT_SELECTION = 5	# After having computed weights for possible advancement provinces using front_battle_province_weight, filter out all but the X top weighted provinces
+	MAX_SIMULTANEOUS_BATTLES = 1	# Allows to spawn up to MAX_SIMULTANEOUS_BATTLES as long as you have that many generals with advance orders.
+	LOCK_PROVINCE_CAPTURE_TO_SINGLE_STATE = no	# If `yes`, provinces will only be captured on the same state as the battle province state.
+	PREFER_BATTLES_IN_UNIQUE_STATES = no	# If `yes`, will avoid creating ANY battles in the same state UNLESS there's no other option.
+}
 
-	NUM_PROVINCES_FOR_ADVANCEMENT_SELECTION = 5		# After having computed weights for possible advancement provinces using front_battle_province_weight, filter out all but the X top weighted provinces
-	MAX_SIMULTANEOUS_BATTLES = 1                    # Allows to spawn up to MAX_SIMULTANEOUS_BATTLES as long as you have that many generals with advance orders.
-	LOCK_PROVINCE_CAPTURE_TO_SINGLE_STATE = no      # If `yes`, provinces will only be captured on the same state as the battle province state.
-	PREFER_BATTLES_IN_UNIQUE_STATES = no            # If `yes`, will avoid creating ANY battles in the same state UNLESS there's no other option.
+NPops = {
+	ASSIMILATION_RATE = 0.003	#0.002 Percentage of pops who will Assimilate into a different Culture each month, if eligible
+	MIN_ASSIMILATION = 30	#10 If there is any assmiliation at least this many people will assimilate
+	CONVERSION_RATE = 0.001	#0.002 Percentage of pops who will Convert into a different Religion each month, if eligible
+	MIN_CONVERSION = 10	# If there is any conversion at least this many people will convert
```

Comment:
I would increase this too,

Reply:
The assimilation or the conversion? I wanted to keep the conversion lower because there aren't even half as many religions in game nad the conversion rate was pretty gamebreakingly fast. The cultures I think I could increase more, but with that new paradox patch do we need it?

Reply:
Min conversion. But you should just check if it still exists in vanilla, because if it doesn't your variable doesn't do anything.

Reply:
I'll check now

Reply:
It still exists, we can increase that number then :)

Reply:
@HiddeLekanne Should be ready for a second review now